### ISSUE TITLE
Date/time misformatting

### ISSUE THREAD

**Issue by jgriffitts** [paiden/Nett#66]
This latest update corrupts date/time entries in my tables. Example program:

```
using System;
using System.Collections.Generic;
using Nett;

namespace ConsoleApp_TOML
{
    class Program
    {
        static void Main(string[] args)
        {
            /* Look for NETT bug with date/time formatting */
            string test_data = @"ActiveSince = 2018-12-26 22:31:33.109707-07:00";
            var test_tbl = Nett.Toml.ReadString(test_data);
            Console.WriteLine(Toml.WriteString(test_tbl));
        }
    }
}
```
When I run this, I get the result:
```
ActiveSince = 2018-12-2622:31:33.109707-07:00
```
The space between date and time has been lost, and the date is not parseable any more.

**Comment by jgriffitts** [comment]
There's something funny about the release version. I cloned the repository and built it locally, and the problem went away. Then I deleted the reference to my local dll and reinstalled it with NuGet, and the problem reappeared.

**Comment by paiden** [comment]
That issue was already fixed on master with 4aba35d8  but is not yet published on NuGet. I hope to publish a new release within the next few days.

**Comment by paiden** [comment]
Fixed with 0.10.1 release
